src/content/docs/products/splashkit/SplashKit Tutorials/Onboarding/01-windows-installation.md

```diff
@@ -0,0 +1,194 @@
+---
+title: Windows Installation Guide
+description: Use this template to create a proposal for a new SplashKit tutorial.
+sidebar:
+  order: 1
+  label: 1. Windows Installation
+  attrs:
+    class: windows
+---
+
+This is a guide on how to install Splashkit on Windows, along with Visual Studio Code and the necessary extensions required to run Splashkit. This guide is intended to get you set up and ready to contribute to the SplashKit team if you're a Windows user. This installation guide was done on Windows 10.
+
+## Install msys2
+
+### Download msys2
+
+To start, head to the official [msys2 website](https://www.msys2.org/) and download the installer.
+
+![msys2](/splashkit/windows-installation-fig1.png)
+
+### Install msys2
+
+When the installer is downloaded, run it, you will get a pop up that looks like the following:
+
+![msys2](/splashkit/windows-installation-fig2.png)
+
+You will need to click `more info` then click run. The following shows the screenshots of each step and the options selected:
+
+
+![msys2](/splashkit/windows-installation-fig4.png)
+
+![msys2](/splashkit/windows-installation-fig5.png)
```

Comment:
A point worth noting from the SplashKit installation guide on the splashkit.io site is:

_It is strongly recommended that you use the default install path, but you can customise the other installation options as needed._

Reply:
Please comment before resolving to understand action (even if none).